db/seeds.rb

```diff
@@ -1,7 +1,12 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+
+# User.destroy_all
+Space.destroy_all
+puts "Cleared space and user records"
+
+# User.create!()
+demo_user = User.last
+
+Space.create!(location: "Erich-Kästner-Str. 11, München", volume: 4, price_per_month: 25, user: demo_user)
+Space.create!(location: "Helene-Mayer-Ring 14, 80809 München",volume: 3, price_per_month: 15, user: demo_user)
+Space.create!(location: "Helene-Mayer-Ring 14, 80809 München", volume: 3, price_per_month: 15, user: demo_user, condition: 'Cannot be used for non-water-proof items', access: 'Accessible by car', description: 'Test description')
+puts "Created three spaces"
```

Comment:
I would probably interpolate the count of the spaces or the name of the last space....